cmd/qliksense/root.go

```diff
@@ -36,6 +37,13 @@ func initAndExecute() error {
 		log.Fatal(err)
 	}
 
+	// create dirs and appropriate files for setting up contexts
+	if len(os.Args) == 1 {
```

Comment:
is this `if` required ? this action will be performed anyways right ? like `setUpPaths()`

Reply:
I did it this way to stop `rootCmd.Execute()` on line# 47  from getting executed when I run: `qliksense config`